src/Microsoft.Health.Dicom.Core/Features/Telemetry/HealthCheckMeter.cs

```diff
@@ -0,0 +1,26 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+public sealed class HealthCheckMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    public HealthCheckMeter()
```

Comment:
As context, I don't even like that these metrics are computed in a health check lol. So I don't think we should make their meter called `HealthCheckMeter`. Perhaps this is part of the `InstanceMeter`?

Reply:
Moved to `InstanceMeter`